source/includes/fact-oplog-background.rst

```diff
@@ -1,7 +1,15 @@
-The :term:`oplog` in the source cluster must be large enough to track
-events that happen during the time it takes to complete the initial
-sync to the destination cluster.
 
-If you anticipate syncing a large data set, or if you plan to pause
-synchronization for an extended period of time, increase the size of the
-replica set ``oplog`` in the source cluster.
+
+Removing older operations from the oplog of the source cluster   
```

Comment:
Consider getting rid of the gerund. 

``mongosync`` applies the operations in the ``oplog`` to the data on the destination cluster. When older operations roll off the oplog on the source cluster, sync fails and ``mongosync`` exits.

Also consider avoiding 'migration' since this applies to sync use cases as well.

Reply:
Fixed